src-plugins/iterativeClosestPoint/vtkICPFilter.cxx

```diff
@@ -143,6 +146,36 @@ void vtkICPFilter::Update()
     TransformFilter2->SetInput(TransformFilter1->GetOutput());
     TransformFilter2->SetTransform(this->ICPTransform);
     TransformFilter2->Update();
+
+    vtkDataSet* outputMesh = TransformFilter2->GetOutput();
+    // catheter point coordinates must be transformed manually
+    vtkFieldData* fieldData = outputMesh->GetFieldData();
+    int nbComponents = 0;
+    for (int j = 0; j < fieldData->GetNumberOfArrays(); ++j)
```

Comment:
this function and this one https://github.com/medInria/medInria-private/pull/187/files are the same. Could it be possible to generalize it in medUtilities, something like that?

Reply:
Done